packages/malloy-vscode/src/extension/webview_views/help_view.ts

```diff
@@ -1,57 +0,0 @@
-/*
- * Copyright 2021 Google LLC
- *
- * This program is free software; you can redistribute it and/or
- * modify it under the terms of the GNU General Public License
- * version 2 as published by the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- */
-
-import * as vscode from "vscode";
-import * as path from "path";
-import { getWebviewHtml } from "../webviews";
-import { HelpMessageType, HelpPanelMessage } from "../message_types";
```

Comment:
Might be worth getting rid of these types in `message_types.ts` since they're no longer needed.